src/plugins/vis_types/timeseries/public/application/components/aggs/moving_average.js

```diff
@@ -188,15 +195,12 @@ export const MovingAverageAgg = (props) => {
               })
             }
           >
-            {/*
-              EUITODO: The following input couldn't be converted to EUI because of type mis-match.
-              Should it be text or number?
-            */}
-            <input
+            <EuiFieldNumber
               className="tvbAgg__input"
-              type="text"
-              onChange={handleNumberChange('window')}
+              onChange={handleNumberChange('window', { isOptional: true })}
```

Comment:
Window is required, not optional

Reply:
Yep, but user should have possibility to clear field so that start type new value. Maybe I should somehow rename this?

Reply:
got it, yes can you change it? It is confusing as it is

Reply:
is `isClearable` okay for you?

Reply:
great!